trakt/calendar.py

```diff
@@ -60,14 +66,23 @@ def _get(self):
     def _build(self, data):
         """Build the calendar"""
         self._calendar = []
-        for episode in data:
-            show = episode.get('show', {}).get('title')
-            season = episode.get('episode', {}).get('season')
-            ep = episode.get('episode', {}).get('number')
-            e_data = {'airs_at': airs_date(episode.get('first_aired')),
-                      'ids': episode.get('episode').get('ids'),
-                      'title': episode.get('episode', {}).get('title')}
-            self._calendar.append(TVEpisode(show, season, ep, **e_data))
+        for cal_item in data:
+            show_data = cal_item.get('show', {})
+            episode = cal_item.get('episode', {})
+            first_aired = cal_item.get('first_aired')
+            season = episode.get('season')
+            ep_num = episode.get('number')
+            extract_ids(show_data)
+            show_data.update(show_data)
+            e_data = {
+                'airs_at': airs_date(first_aired),
+                'ids': episode.get('ids'),
+                'title': show_data.get('title'),
```

Comment:
This might be considered a breaking change. But pytrakt was leaving out TV show information. And I really need this.

Reply:
Interesting... Does this actually do what you want it to do? It seems like this would result in a `TVEpsiode` with a title of the show and not the episode

Reply:
Should this line be `'title': episode.get('title')` since this is building up the dict for the `TVEpisode`?

Reply:
Should I remove it? it doesn't have a purpose for my usecase